server/plugins/task-plugin-api/src/main/java/com/oceanbase/odc/plugin/task/api/datatransfer/dumper/ExportOutput.java

```diff
@@ -102,6 +107,17 @@ public void toFolder(@NonNull File folder) throws IOException {
         if (!folder.exists()) {
             FileUtils.forceMkdir(folder);
         }
+
+        if (inputDir != null) {
+            // if input file is a dir, try to use symbolic link, avoid copy files if possible
+            try {
+                Files.createSymbolicLink(this.inputDir, folder.toPath());
```

Comment:
when the symbolic link will be unlinked ?

is it possibile to read from from origin directory?

Reply:
after data transfer task finished, either success or failure, the /datatransfer/${seq}  temp directory will be cleaned, and files will be unlinked.

if read from origin directory, log files and some hidden files will be written to that place, which might cause some trouble for user.